src/Button/Button.tsx

```diff
@@ -45,7 +45,8 @@ type Props = {
   type?: 'button' | 'submit' | 'reset'
 }
 
-export type ButtonProps = Props & ButtonHTMLAttributes<HTMLButtonElement>
+export type ButtonProps = Props &
+  Omit<ButtonHTMLAttributes<HTMLButtonElement>, 'color'>
```

Comment:
`ButtonHTMLAttributes<HTMLButtonElement>` have a `color` type so it was overriding the one specified in `Props`